src/users/users.ts

```diff
@@ -82,6 +85,24 @@ export class Users {
     return deserializeUser(data);
   }
 
+  async authenticateUserWithMagicAuth(
+    payload: AuthenticateUserWithMagicAuthOptions,
+  ): Promise<AuthenticationResponse> {
+    const { data } = await this.workos.post<
+      AuthenticationResponseResponse,
```

Comment:
Apologies if that has been discussed elsewhere already, but have we thought about alternative names for this interface?

Reply:
I'd prefer to keep it consistent across the board. We have `Entity` and `EntityResponse`. It's not ideal, but I think being inconsistent would be worse.